MathLib/LinAlg/LinAlg.h

```diff
@@ -148,6 +148,10 @@ namespace LinAlg
 
 // Vector
 
+/// Set local accessible vector in order to get entries.
+/// Call this before call operator[] or get(...).
+void setLocalAccessibleVector(PETScVector const& x);
```

Comment:
Kept this comment because the member is real, while improved the documentation of  setLocalAccessibleVector  in LinAlg.h.